src/model/asset.js

```diff
@@ -21,7 +21,7 @@ class Asset {
                 }
 
                 if (query === '') {
```

Comment:
Please double check the condition, behaviour is now inconsistent.

- when clicking on the images tab, 999 assets are listed (so far so good, reported issue is fixed)
- when using search, 25 assets are listed (adding the message from [other branch](https://github.com/Frontify/sketch/commit/4399fca3489ecbd801d36d16190a1e98adec5184)) will help here.
- when deleting the search term, 25 assets are listed. I was expecting 999 (let's fix that by adjusting the condition)
- when clicking the images tab again, there are 999 assets listed

Reply:
Thanks for solving this, I was wondering how the 25 assets can get there!

Reply:
Fixed it. If you use search an then delete id, query is `q=`, which to the backend means there is a query (although empty) and it only returns 25 assets. I set the query to empty string if it equals `q=`, that solves the issue.